websites/D/dizipal/presence.ts

```diff
@@ -0,0 +1,147 @@
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, href } = document.location,
+		seriesPageCheck =
+			document.querySelector(
+				"#container > div.popup-inner.auto > div.cover > div > a:nth-child(1)"
+			) === null;
+
+	for (const [key, value] of Object.entries(pages)) {
+		if (pathname.includes("/dizi/") && seriesPageCheck) {
+			const stringsData = await strings,
+				seriesTitle = document
+					.querySelector("#container")
+					.querySelector("h5")?.textContent,
+				episode = document.querySelector(
+					"#container > div.episode-head > div:nth-child(1) > header > h2 > a"
+				)?.textContent,
+				seriesUrl = document.querySelector<HTMLAnchorElement>(
+					"#container > div.episode-head > div:nth-child(1) > header > h2 > a"
+				)?.href;
+
+			if (seriesTitle && !episode) {
+				presenceData.details = `${seriesTitle} Adlı Diziye Göz Atıyor.`;
+
+				presenceData.buttons = [
+					{
+						label: `${stringsData.viewShow} ${seriesTitle}`,
+						url: href,
+					},
+				];
+			} else if (episode) {
+				delete presenceData.startTimestamp;
+
+				if (video) {
+					presenceData.details = `${episode} İzliyor.`;
+
+					presenceData.state = `${
+						document.querySelector(
+							"#container > div.episode-head > div:nth-child(1) > h6"
+						)?.textContent
+					} ${video.paused ? stringsData.pause : stringsData.play}`;
+
+					[presenceData.startTimestamp, presenceData.endTimestamp] =
+						presence.getTimestamps(video.currentTime, video.duration);
+				} else presenceData.details = `${episode} Adlı Bölüme Göz Atıyor.`;
+
+				presenceData.buttons = [
+					{
+						label: stringsData.viewEpisode,
+						url: href,
+					},
+					{
+						label: "Diziye Git",
+						url: seriesUrl,
+					},
+				];
+			}
+		} else if (
+			document.querySelector("#pre_content > div.g-title > div > span")
+				?.textContent
+		) {
+			const stringsData = await strings;
+
+			presenceData.details = document.querySelector(
+				"#pre_content > div.g-title > div > span"
+			)?.textContent;
+
+			if (video?.currentTime && video?.duration) {
+				presenceData.state = video.paused
+					? stringsData.pause
+					: stringsData.play;
+
+				[presenceData.startTimestamp, presenceData.endTimestamp] =
+					presence.getTimestamps(video.currentTime, video.duration);
+
+				presenceData.smallImageKey = video.paused ? Assets.Pause : Assets.Play;
+				presenceData.smallImageText = video.paused
+					? stringsData.pause
+					: stringsData.play;
+			}
+
+			presenceData.buttons = [
+				{
+					label: stringsData.viewMovie,
+					url: href,
+				},
+			];
+		} else if (pathname.includes("/koleksiyon/")) {
+			presenceData.details = `${
+				document.querySelector(".g-title > div")?.textContent
+			} Koleksiyonuna Göz Atıyor`;
+		} else if (pathname.includes("@")) {
+			presenceData.details = `${
+				document.querySelector(".title")?.textContent
+			} Adlı Kullanıcının Profiline Göz Atıyor`;
+		} else if (pathname === key) presenceData.details = value;
+	}
```

Comment:
So this only runs it if no other if goes through 2/2
```suggestion
} else {
		for (const [key, value] of Object.entries(pages))
			if (pathname === key) presenceData.details = value;
	}
```